migtests/tests/pg-dvdrental/init-db

```diff
@@ -19,5 +19,8 @@ run_psql postgres "CREATE DATABASE ${SOURCE_DB_NAME};"
 echo "Initialising source database."
 run_pg_restore ${SOURCE_DB_NAME} dvdrental.tar
 
+echo "Adding custom collation."
+run_psql ${SOURCE_DB_NAME} "CREATE COLLATION fulah (provider= icu, locale='ff-x-icu');"
```

Comment:
Note that I have chosen a collation which is not provided by `libc`, YB only supports C for the libc provider, which is the default collation used